wremnants/histselections.py

```diff
@@ -843,6 +843,7 @@ def calculate_fullABCD_smoothed(self, h, syst_variations=False, use_spline=False
             svar *= 1./xwidth**2
 
             goodbin = (sval > 0.) & (svar > 0.)
+            goodbin = goodbin & ~((sval < 1.) & (svar/sval**2 <= 1.)) # exclude bins with 0 data entries but negative prompt MC subtraction
```

Comment:
It would probably be OK but it would be a stronger interference and biasing the number of events in one direction. I would wait to see if it's already enough what we get after the last commit, maybe it's already enough